### ISSUE TITLE
Multiline comment inserts extra # > even if I already have closing tag

### ISSUE THREAD

**Issue by AdamLeMmon01** [ant-druha/intellij-powershell#252]
When I add a multiline PowerShell comment, and then add a new line in the middle, an extra character is added automatically.

I end up with this:
```
<#
    .SYNOPSIS
        Some text
    # > # <- this keeps getting added each time I try to add a new line
    # >
#>

**Comment by ForNeVeR** [comment]
I think this is a duplicate of #183. That is already closed, ~~but the fix is not yet released; I am preparing some final patches for the upcoming release. If everything goes as planned, it should be available in a few days.~~

**Comment by ForNeVeR** [comment]
But now wait a minute. The fix is supposed to be in 2.5.0, already released. What plugin version do you use?

**Comment by AdamLeMmon01** [comment]
I updated to the latest plugin version today and this appears to be fixed. Thanks. We can close this out
